test/specs/templates.e2e.ts

```diff
@@ -27,6 +27,220 @@ describe('Templates', async () => {
     projectName = testSetup.tempProjectName.toUpperCase();
   });
 
+  // Apex Class
+  step('Create an Apex Class', async () => {
+    // Using the Command palette, run SFDX: Create Apex Class.
+    const workbench = await browser.getWorkbench();
+    const inputBox = await utilities.runCommandFromCommandPrompt(workbench, 'SFDX: Create Apex Class', 1);
+
+    // Set the name of the new component to ApexClass1.
+    await inputBox.setText('ApexClass1');
+    await inputBox.confirm();
+    await utilities.pause(1);
+
+    // Select the default directory (press Enter/Return).
+    await inputBox.confirm();
+
+    const successNotificationWasFound = await utilities.attemptToFindNotification(workbench, 'SFDX: Create Apex Class successfully ran', 10);
+    expect(successNotificationWasFound).toBe(true);
+
+    const outputPanelText = await utilities.attemptToFindOutputPanelText('Salesforce CLI', 'Finished SFDX: Create Apex Class', 10);
```

Comment:
I think the output validations that are in pushAndPull would be good here instead

Reply:
done